src/content/en/updates/2018/12/badging-api.md

```diff
@@ -61,54 +68,68 @@ Examples of sites that may use this API include:
 | Step                                       | Status                       |
 | ------------------------------------------ | ---------------------------- |
 | 1. Create explainer                        | [Complete][explainer]        |
-| **2. Create initial draft of specification** | [**In Progress**][spec]    |
+| 2. Create initial draft of specification   | [Complete][spec]             |
 | **3. Gather feedback & iterate on design** | [**In progress**](#feedback) |
-| 4. Origin trial                            | Not started                  |
+| **4. Origin trial**                        | [**In progress**](#ot)       |
 | 5. Launch                                  | Not started                  |
 
+
 ## How to use the badging API {: #use }
 
-Dogfood: We are still iterating on the design of the Badging API, and it’s
-**not** available in the browser yet. The sample code you see is based on
-the current design, and will likely change between now and the time it lands
-in the browser.
+Starting in Chrome 73, the Badging API is available as an origin trial
+for (?PLATFORMS?).
```

Comment:
What platforms will this be available on?

Reply:
Just Windows (7+) and OSX.